src/bot/commands/random/dog.ts

```diff
@@ -0,0 +1,214 @@
+import { Command, Input, Listener } from '@api';
+import * as request from 'request';
+import { Response } from 'request';
+import { Message } from 'discord.js';
+import { RichEmbed } from 'discord.js';
+import { TextChannel } from 'discord.js';
+import { Emoji } from '@bot/libraries/emoji';
+import { Reactions } from '@bot/libraries/reactions';
+import { Framework } from '@core/framework';
+import { Timer } from '@bot/libraries/utilities/timer';
+import { ReactionListener } from '@bot/listeners/reactions';
+import { listeners } from 'cluster';
+import { GuildMember } from 'discord.js';
+import { Economy } from "@libraries/economy";
+import { debug } from 'util';
+import { Url } from 'url';
+const entities = require("html-entities").AllHtmlEntities;
+
+let breeds: string[] = [
```

Comment:
Type `string[]` is assumed.

```suggestion
let breeds = [
```